components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/pom.xml

```diff
@@ -151,6 +151,11 @@
             <artifactId>org.wso2.carbon.apimgt.impl</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
can u check whether we need to add **provided** scope to this? Otherwise wouldn't the war bundle the grohQL jar as well?

Reply:
yes. fixed